system/jlib/jdebug.cpp

```diff
@@ -1650,8 +1651,18 @@ class CExtendedStats  // Disk network and cpu stats
 
     StringBuffer ifname;
 
+#ifdef __linux__
+    UnsignedArray majmin;
+#endif
+
     int isdisk(unsigned int major, unsigned int minor)
     {
+#ifdef __linux__
+        unsigned mm = (major<<16)+minor;
+        bool found = majmin.contains(mm);
```

Comment:
when isDisk changed to return bool, can just be : 
```
    return majmin.contains(nm);
```

Reply:
I did this specifically so that if lsblk cmd failed then it keeps trying the old way.

Reply:
yes, understood, makes sense.